src/plinder/core/structure/structure.py

```diff
@@ -544,7 +545,7 @@ def protein_calpha_coords(self) -> NDArray[np.double]:
     def input_ligand_conformers(self) -> dict[str, Chem.Mol]:
         """Ligand 3D mol conformer objects from input SMILES"""
         assert self.ligand_mols is not None
-        return {tag: mol_tuple[1] for tag, mol_tuple in self.ligand_mols.items()}
+        return {tag: mol_tuple[0] for tag, mol_tuple in self.ligand_mols.items()}
```

Comment:
mol_tuple[0] is only 2D RDKit mol made from smiles, for conformers - it would be mol_tuple[1]